test/scripts/forked_state/docker-compose.yaml

```diff
@@ -1,21 +1,28 @@
 services:
   anvil:
```

Comment:
this should all go into `test/scripts/7702`

Reply:
Why? The 7702 infra is compatible with the not. Separating these will make testing more complicated as you have to run only specific tests against one or the other environment vs just running all available